### PR TITLE
fix WATCH_NAMESPACE env variable in container spec of generated CSV

### PR DESCRIPTION
**Pull request by MoserMichael**
fix for https://bugzilla.redhat.com/show_bug.cgi?id=1840220
WATCH_NAMESPACE doesn't get initialised properly

### CODE REVIEW

File: manifests/node-maintenance-operator/v0.6.0/node-maintenance-operator.v0.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -126,7 +126,7 @@ spec:
                   value: ""
```
Comment (omeryahud) [inline manifests/node-maintenance-operator/v0.6.0/node-maintenance-operator.v0.6.0.clusterserviceversion.yaml:126]: The issue still remains, `value` and `valueFrom` are mutually exclusive.

Comment (MoserMichael) [reply manifests/node-maintenance-operator/v0.6.0/node-maintenance-operator.v0.6.0.clusterserviceversion.yaml]: Correct! pushed a fix for that now.

**Comment by beekhof** [comment]
Do we know which yaml file (or tool) HCO is using for constructing our deployments?

/approve
/lgtm

**Pull request merged**